examples/svelte/hydration/src/routes/+page.ts

```diff
@@ -0,0 +1,11 @@
+import { dehydrate, QueryClient } from '@tanstack/svelte-query'
+import { getPosts, limit } from '$lib/data.js'
+
+/** @type {import('./$types').PageLoad} */
+export async function load() {
+  const queryClient = new QueryClient()
+  await queryClient.prefetchQuery(['posts', limit], () => getPosts(limit))
```

Comment:
it would be good if all examples were to use the object syntax already, like:

```
queryClient.prefetchQuery({
  queryKey: ['posts', limit],
  queryFn: () => getPosts(limit)
})
```

this will be the only valid syntax in v5, and we've done this for all docs and examples of the other frameworks already.